### PR TITLE
pull in cqm-models v4.1.2

### PR DESCRIPTION
**Pull request by adongare**
This PR has following changes:
- pull cqm-models v4.1.2
- fixed https://oncprojectracking.healthit.gov/support/browse/BONNIEMAT-1096, results of aggregate functions ignored if it is 0
- few Security vulnerabilities 

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by adongare** [comment]
Hi @dczulada, can you please review this? this pulls in cqm-models v4.1.2 to provide CQM `is` operator suport for entities

**Review by jkotanchik-SB — DISMISSED** [review]

File: lib/helpers/results_helpers.js [hunk 1]
```diff
@@ -462,7 +463,7 @@ module.exports = class ResultsHelpers {
       prettyResult += `${currentIndentation}}`;
       return prettyResult;
     }
-    if (result) {
+    if (result || result === 0) {
```
Comment (adongare) [inline lib/helpers/results_helpers.js:466]: Results of aggregate functions being ignored if it is 0(https://oncprojectracking.healthit.gov/support/browse/BONNIEMAT-1096)

**Review by jkotanchik-SB — APPROVED** [review]

**Review by RohitKandimalla — APPROVED** [review]

**Comment by adongare** [comment]
I'm ignoring the vulnerability with word-wrap as there is no fix/patch available and it is dev dependency.
